debian/control

```diff
@@ -2,14 +2,17 @@ Source: nnstreamer-ros
 Section: libs
 Priority: optional
 Maintainer: MyungJoo Ham <myungjoo.ham@samsung.com>
-Build-Depends: gcc (>=5), cmake, debhelper (>=9),
+Build-Depends: bsdmainutils, gcc (>=5), cmake, debhelper (>=9),
  libgstreamer1.0-dev, libgstreamer-plugins-base1.0-dev, libglib2.0-dev,
  gstreamer1.0-plugins-base, gstreamer1.0-tools,
  libgtest-dev, ssat, python2.7, nnstreamer-dev,
- ros-kinetic-catkin, ros-kinetic-cmake-modules,
- ros-kinetic-genmsg, ros-kinetic-message-generation,
- ros-kinetic-roscpp, ros-kinetic-rosbag
-Standards-Version: 3.9.7
+ ros-kinetic-catkin | ros-melodic-catkin | ros-noetic-catkin,
+ ros-kinetic-cmake-modules | ros-melodic-cmake-modules | ros-noetic-cmake-modules,
+ ros-kinetic-genmsg | ros-melodic-genmsg | ros-noetic-genmsg,
+ ros-kinetic-message-generation | ros-melodic-message-generation | ros-noetic-message-generation,
+ ros-kinetic-roscpp | ros-melodic-roscpp | ros-noetic-roscpp,
+ ros-kinetic-rosbag | ros-melodic-rosbag | ros-noetic-rosbag,
+Standards-Version: 4.5.0
```

Comment:
Please describe why you need to update the debian standards version.

Reply:
dh_lintian complains about it. That is the reason why I used the up-to-date version.

Reply:
If 4.5.0 is supported by Ubuntu 18.04, it is fine. I'm worried because 4.5.0 is released in 2020, so I'm not sure if it is supported by Ubuntu 18.04.

Reply:
@myungjoo In my acknowledge, it is just the most recent version of [this document](https://www.debian.org/doc/debian-policy/ch-source.html#s-standardsversion)